python-main.js

```diff
@@ -85,6 +85,8 @@ function pythonEditor(id, autocompleteApi) {
     editor.enableAutocomplete = function(enable) {
         ACE.setOption('enableBasicAutocompletion', enable);
         ACE.setOption('enableLiveAutocompletion', enable);
+        if(!ACE.completer){ACE.execCommand('startAutocomplete');
```

Comment:
Apologies for the nitpicking, but could we format these lines as follows?

```js
if (!ACE.completer) {
    ACE.execCommand('startAutocomplete');
    editor.triggerAutocompleteWithEnter(false);
}
```

It's good to keep a constant style in the same code file, so if you have a look at the code around your changes, it should be a good indication on how to style it.

Reply:
Actually, this check is already performed inside triggerAutocompleteWithEnter(), so you should be able to replace all that with only `editor.triggerAutocompleteWithEnter(false);` no?